lib/analyzer/index.ts

```diff
@@ -18,27 +18,28 @@ async function analyze(
   dockerfileAnalysis?: dockerFile.DockerFileAnalysis,
   options?: DockerOptions,
 ) {
-  const [imageInspection, osRelease] = await Promise.all([
-    imageInspector.detect(targetImage, options),
-    osReleaseDetector.detect(targetImage, dockerfileAnalysis, options),
-  ]);
-
-  const pkgPaths = [...aptAnalyzer.APT_PKGPATHS, ...apkAnalyzer.APK_PKGPATHS];
-
   const docker = new Docker(targetImage, options);
-  let pkgFiles: ExtractedKeyFiles;
 
-  try {
-    pkgFiles = await docker.extract(pkgPaths);
-  } catch (err) {
-    debug(err);
-    throw new Error(err);
-  }
+  await docker.maybeStaticScan(
```

Comment:
can you please replace `maybeStaticScan` with something like `scanStaticalyIfNeeded`?